pkg/virt-launcher/virtwrap/manager.go

```diff
@@ -650,13 +650,11 @@ func (l *LibvirtDomainManager) generateConverterContext(vmi *v1.VirtualMachineIn
 	diskInfo := make(map[string]*containerdisk.DiskInfo)
 	for i, volume := range vmi.Spec.Volumes {
 		if volume.VolumeSource.PersistentVolumeClaim != nil {
-			isBlockPVC := false
 			if _, ok := hotplugVolumes[volume.Name]; ok {
-				isBlockPVC = isHotplugBlockDeviceVolume(volume.Name)
+				isBlockPVCMap[volume.Name] = isHotplugBlockDeviceVolume(volume.Name)
```

Comment:
Did you take this from https://goreportcard.com/report/github.com/kubevirt/kubevirt? I think their ineffassign version is an older version with a bug. As you can see this is clearly not a code issue. When you type `make`, latest `ineffassign` checks are performed in parallel to the build.